### DISCUSSION TITLE
Meeting: March 5th, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#87] [Announcements]
**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).

![image](https://github.com/OAI/sig-moonwalk/assets/54582/09ac7e12-59cd-4061-a7ec-1e36bb6387a7)

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |

**Comment by earth2marsh** [comment]
@handrews 's PR #86 , covering IRIs (and the first use of our ADR approach!)

**Reply by earth2marsh** [reply]
Reviewed and merged!
The Chinese example seems to translate as "example" and "china". 
I'm not sure what the Arabic example says, but guessing it is similar? Would be good to check.

**Reply by handrews** [reply]
@earth2marsh to close the loop on this, I finally remembered that I got the Arabic one from https://www.w3.org/International/iri-edit/BidiExamples so I rather assume it's innocuous.  Dumping the components into Google Translate got all sorts of weirndess as it guessed different langauges for each.  I think I made the Chinese one by putting "example" and "China" into Google translate.

**Comment by earth2marsh** [comment]
Resource orientation discussion from #30 .

**Comment by earth2marsh** [comment]
Also discussed this discussion agenda template and how to automate. @darrelmiller confirmed that the OAS repo uses a GitHub action to do this, and @ralfhandl pointed to https://github.com/marketplace/actions/create-github-discussion

**Comment by earth2marsh** [comment]
OH: "Implying semantics from the URI resource is an explicitly opt-in feature."

**Comment by earth2marsh** [comment]
Recording here: https://zoom.us/rec/share/qMFvdLo4RQ2lhmbQ6U6N2lF12mpoELMkpHSJMezn1uBBsKMDgNKjmGI18T99UIMQ.uaI8UjgUrTQM7pg3
